site/docs/latest/overview/overview.md

```diff
@@ -29,6 +29,8 @@ BookKeeper is suitable for a wide variety of use cases, including:
 Use case | Example
 :--------|:-------
 [WAL](https://en.wikipedia.org/wiki/Write-ahead_logging) (write-ahead logging) | The HDFS [namenode](https://hadoop.apache.org/docs/r2.5.2/hadoop-project-dist/hadoop-hdfs/HDFSHighAvailabilityWithNFS.html#BookKeeper_as_a_Shared_storage_EXPERIMENTAL)
+[WAL](https://en.wikipedia.org/wiki/Write-ahead_logging) (write-ahead logging) | Twitter [Manhattan](https://blog.twitter.com/engineering/en_us/a/2014/manhattan-our-real-time-multi-tenant-distributed-database-for-twitter-scale.html)
```

Comment:
this one is probably better: https://blog.twitter.com/engineering/en_us/a/2016/strong-consistency-in-manhattan.html

or maybe just put two links there.

Reply:
"maybe just put two links there"

I am not sure about the layout with two links

okay for changing to the link you pointed